cytotable/presets.py

```diff
@@ -92,6 +92,54 @@
                 AND per_nuclei.Nuclei_Number_Object_Number = per_cytoplasm.Cytoplasm_Parent_Nuclei
             """,
     },
+    "cellprofiler_sqlite_pycytominer": {
```

Comment:
love this!

Could it also be worth specifying `"CELLPROFILER_VERSION"` as a field? I think some of the identifying columns may be different. CC'ing @jenna-tomkinson to hear her thoughts on any differences she may forsee according to CellProfiler version

Reply:
I would hope that CellProfiler does not change the naming conventions of the columns anytime in the future. It would be very unnecessary unless there was a big reason why it would need to be changed.

It might be good to add the CellProfiler version in general just in case.

Reply:
Thank you both! I like this idea and will work towards incorporating it in.

Reply:
I've added this data into the configuration presets. In the case of the configuration preset used for this PR's addition, I felt that we needed to reference both CellProfiler and Pycytominer's versions, so the new key is itself a dictionary of multiple elements to help keep this flexible. Note: I also intend to add more comprehensive schema details for input and output in #27, which I feel will complement this work.